packages/config/config/devices/0x008b/trane_xr524.json

```diff
@@ -235,20 +235,30 @@
 			"defaultValue": 80
 		},
 		"75": {
-			"label": "Fan ON Time",
+			"label": "Manually Run Fan",
 			"unit": "minutes",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 120,
 			"defaultValue": 0
 		},
 		"76": {
-			"label": "Fan OFF Time",
-			"unit": "minutes",
+			"label": "Run/Hold Schedule",
 			"valueSize": 1,
 			"minValue": 10,
 			"maxValue": 120,
-			"defaultValue": 10
+			"defaultValue": 10,
```

Comment:
I think this should be 1 not 10.

Reply:
Which? The min value or the default?

Reply:
I meant the default value but now that you mention it, since 0 and 1 are the only valid values, that propbably should be the min and max as well, unless they are not needed when there are option choices. Not overly famillar with the config format.

Reply:
Yeah, there's something off about this parameter definition